via/static/scripts/video_player/components/test/ToastMessages-test.js

```diff
@@ -158,9 +136,23 @@ describe('ToastMessages', () => {
 
     animationContainer
       .getDOMNode()
-      .dispatchEvent(
-        new AnimationEvent('animationend', { animationName: 'invalid' })
-      );
+      .dispatchEvent(new AnimationEvent('animationend'));
+
+    assert.called(fakeOnMessageDismiss);
+  });
+
+  it('does not invoke onTransitionEnd for animations on invalid target element', () => {
```

Comment:
The meaning of "invalid" could be clearer here. You mean a child of the toast message container, right?

Reply:
Yeah. I just changed it to `does not invoke onTransitionEnd for animation events bubbling from children`